Doc/whatsnew/3.10.rst

```diff
@@ -273,6 +273,13 @@ Added *globalns* and *localns* parameters in :func:`~inspect.signature` and
 local and global namespaces.
 (Contributed by Batuhan Taskaya in :issue:`41960`.)
 
+ipaddress
+---------
+
+XXX Add :func:`_BaseNetwork.next_network` to support finding the next closest
```

Comment:
```suggestion
Added :func:`_BaseNetwork.next_network` to support finding the next closest
```

Reply:
I would recommend mentioning the concrete classes, not the private base class.
(but that’s details for later in the review process)

Reply:
Thank you both for your review, I have removed the `XXX` part and referred to the IPv4Network and IPv6Network concrete classes instead of the private base class.  I have also reworded this description slightly.